packages/plugin-core/src/test/suite-integ/NoteLookupCommand.test.ts

```diff
@@ -749,6 +753,430 @@ suite("NoteLookupCommand", function () {
       });
     });
 
+    describeMultiWS(
+      "WHEN schema template references to a template note that lies in a different vault",
+      {
+        ctx,
+        preSetupHook: async ({ wsRoot, vaults }) => {
+          await ENGINE_HOOKS_MULTI.setupBasicMulti({ wsRoot, vaults });
+          // Template is in vault2
+          await NoteTestUtilsV4.createNote({
+            wsRoot,
+            body: "food ch2 template",
+            fname: "template.ch2",
+            vault: vaults[1],
+          });
+        },
+        postSetupHook: async ({ wsRoot, vaults }) => {
+          // Schema is in vault1
+          const vault = vaults[0];
+          await NoteTestUtilsV4.createSchema({
+            fname: "food",
+            wsRoot,
+            vault,
+            modifier: (schema) => {
+              const schemas = [
+                SchemaUtils.createFromSchemaOpts({
+                  id: "food",
+                  parent: "root",
+                  fname: "food",
+                  children: ["ch2"],
+                  vault,
+                }),
+                SchemaUtils.createFromSchemaRaw({
+                  id: "ch2",
+                  template: {
+                    id: "template.ch2",
+                    type: "note",
+                  },
+                  namespace: true,
+                  vault,
+                }),
+              ];
+              schemas.map((s) => {
+                schema.schemas[s.id] = s;
+              });
+              return schema;
+            },
+          });
+        },
+      },
+      () => {
+        test("THEN template body gets applied to new note", async () => {
```

Comment:
can you make this more descriptive? "THEN template body gets applied to new note FROM other vault"